.github/workflows/ui-translations.yml

```diff
@@ -0,0 +1,55 @@
+name: FOLIO UI Compile Translations
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        description: Node version to use, in semver notation
+        required: true
+        type: string
+      folio-npm-registry:
+        description: Full URL to the registry to use for @folio modules
+        required: true
+        type: string
+
+jobs:
+  run:
+    name: Compile translations
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          show-progress: false # spammy
+
+      - name: Install Node ${{ inputs.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ inputs.node-version }}
+          check-latest: true
+          always-auth: true
+
+      - name: Set FOLIO NPM registry
+        run: yarn config set @folio:registry ${{ inputs.folio-npm-registry }}
+
+      - id: get-version
+        name: Get current @formatjs/cli version
+        run: echo version=$(jq -r '.devDependencies["@formatjs/cli"]' package.json)
+
+      # we do it this way to avoid installing all modules
+      - name: Install @formatjs/cli
+        run: |-
+          rm package.json
+          yarn add @formatjs/cli@${{ steps.get-version.outputs.version }}
+          git checkout package.json
```

Comment:
Couple things:
1. do we have to indicate the `jq` dependency somewhere or is it really ok to rely on it being present?
2. alternatively/preferably, translation compilation is now built directly into stripes-cli so individual apps no longer have to (a) depend on `@formatjs/cli` or (b) include their own `formatjs-compile` script. The command is `stripes translate compile`. Can we use that here? We may still be able to leverage this `rm package.json` hack and just install `@folio/stripes-cli`, dunno.

Reply:
1. Yeah, we get a standard ubuntu 2? something docker env which includes it
2. That would be possible, didn't know stripes-cli did that

Reply:
Remember, I compiled my first program in the PREVIOUS MILLENNIUM when disk was expensive and nothing was installed by default. If `jq` is standard fare, great.

Reply:
Ah yep, we specifically have `jq`: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md?plain=1#L85

Reply:
<img width="955" alt="image" src="https://github.com/folio-org/.github/assets/8005215/259fd8c7-3d7f-47e6-894e-96ea8740327c">
Looks like it's working!